packages/bundle/src/renderMarkdown.js

```diff
@@ -7,7 +7,7 @@ import sanitizeHTML from 'sanitize-html';
 
 const SANITIZE_HTML_OPTIONS = {
   allowedAttributes: {
-    a: ['aria-label', 'href', 'name', 'target', 'title'],
+    a: ['aria-label', 'href', 'name', 'target', 'title', 'role', 'class'],
```

Comment:
I don't think that 'class' belongs in this list. As my understanding is that these are supported markdown attributes and we don't want the markdown author to support overriding this behavior. 

It also looks like role was added as part of this change, but not necessarily used. It might be left over from another change you inherited.